zingolib/src/wallet/transactions.rs

```diff
@@ -4,45 +4,41 @@ use std::{
 };
 
 use byteorder::{LittleEndian, ReadBytesExt, WriteBytesExt};
-use incrementalmerkletree::witness::IncrementalWitness;
+use incrementalmerkletree::Position;
```

Comment:
A `Position` is where a leaf is..  so it's a scalar that indexes into an ordered structure.